README.md

```diff
@@ -38,6 +38,11 @@ to read secrets, which can both be used side-by-side:
 
 ## Getting Started
 
+The [learn guide][vault-learn-guide] is the most complete and fully explained
```

Comment:
This looks like an error. Should it be `[learn guide](vault-learn-guide)`?

Reply:
Not an error, I just double checked and the link does work. This is an alternative way to specify links in markdown. `vault-learn-guide` references a label for a URL at the bottom of this document.

Reply:
Cool, thanks!

Reply:
BTW the reason I am using it is mainly as a nice way reference the same link multiple times, but I also like how it means all the links can get grouped together and they keep the noise out of the main text of the markdown source.